### PR TITLE
Add new RPC status and error code

### PR DESCRIPTION
**Pull request by bufrr**
### Proposed changes in this pull request
Explain the changes in this pull request in order to help the project maintainers understand the overall impact of it.

### Type (put an `x` where ever applicable)
- [ ] Bug fix: Link to the issue
- [ ] Feature (Non-breaking change)
- [ ] Feature (Breaking change)
- [ ] Documentation Improvement

### Checklist
Please put an `x` against the checkboxes. Write a small comment explaining if its `N/A` (not applicable)

- [ ] Read the [CONTRIBUTION guidelines](https://github.com/nknorg/nkn#contributing).
- [ ] All the tests are passing after the introduction of new changes.
- [ ] Added tests respective to the part of code I have written.
- [ ] Added proper documentation where ever applicable (in code and README.md).
- [ ] Code has been written according to [NKN-Golang-Style-Guide](https://github.com/nknorg/nkn/wiki/NKN-Golang-Style-Guide)

### Extra information
Any extra information related to this pull request.

### CODE REVIEW

File: api/httpjson/RPCserver.go [hunk 1]
```diff
@@ -254,6 +258,9 @@ func (s *RPCServer) SetLocalNode(ln *node.LocalNode) {
 func (s *RPCServer) GetNetNode() (*node.LocalNode, common.ErrorWithCode) {
 	ln := s.GetLocalNode()
 	if ln == nil {
+		if s.GetLedgerStatus() == false && config.Parameters.StatePruningMode == "lowmem" {
```
Comment (yilunzhang) [inline api/httpjson/RPCserver.go]: We don't need `config.Parameters.StatePruningMode == "lowmem"` here

Comment (bufrr) [reply api/httpjson/RPCserver.go]: fixed

File: nknd.go [hunk 2]
```diff
@@ -196,18 +196,24 @@ func nknMain(c *cli.Context) error {
...
+	rpcServer := httpjson.NewServer(nil, wallet)
+	go rpcServer.Start()
+
 	// initialize ledger
 	err = InitLedger(account)
 	if err != nil {
 		return fmt.Errorf("chain.initialization error: %v", err)
 	}
 
+	rpcServer.SetLedgerStatus(true)
```
Comment (yilunzhang) [inline nknd.go]: It's better to put this line after defer below so if this function call fails, the store will still be closed

Comment (bufrr) [reply nknd.go]: fixed

**Review by yilunzhang — COMMENTED** [review]
Looks good to me

**Pull request merged**